apps/wallet/src/ui/app/pages/accounts/manage/accounts-finder/AccountsFinderIntroPage.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { useNavigate } from 'react-router-dom';
+import { Overlay } from '_components';
+import { useActiveAccount } from '_hooks';
+import { getKey } from '_helpers';
+import { Theme, useTheme } from '@iota/core';
+import { Button, ButtonType } from '@iota/apps-ui-kit';
+import BalanceFinderIntroImage from '_assets/images/balance_finder_intro.svg';
+import BalanceFinderIntroDarkImage from '_assets/images/balance_finder_intro_darkmode.svg';
+
+export function AccountsFinderIntroPage() {
+    const { theme } = useTheme();
+    const navigate = useNavigate();
+    const activeAccount = useActiveAccount();
+
+    const accountSourceId = activeAccount && getKey(activeAccount);
```

Comment:
we should probably consider ledger here too because this screen is used by ledger wallets